dom/README.md

```diff
@@ -52,7 +52,7 @@ the app on the DOM.
 
 - `container: String|HTMLElement` the DOM selector for the element (or the element itself) to contain the rendering of the VTrees.
 - `options: DOMDriverOptions` an object with two optional fields: `transposition: boolean` enables/disables transposition of inner streams in
-the virtual DOM tree, `modules: array` contains additional Snabbdom modules.
+the virtual DOM tree, `modules: array` overrides the default Snabbdom modules.
```

Comment:
How about a link where I can see what the default Snabbdom modules are?
And it should be also clear, whether these defaults are in Cycle.js or upstream Snabbdom defaults.